cerbero/packages/wix.py

```diff
@@ -571,3 +582,66 @@ def _add_start_menu_shortcuts(self):
     def _add_vs_properties(self):
         etree.SubElement(self.product, 'PropertyRef', Id='VS2010DEVENV')
         etree.SubElement(self.product, 'PropertyRef', Id='VC2010EXPRESS_IDE')
+
+class Burn(WixBase):
+    '''Creates an installer package from a
+       L{cerbero.packages.package.Package} with a burn bundle
+    '''
+    def __init__(self, config, package, packages_deps, wix_config, store, paths):
+        WixBase.__init__(self, config, package)
+        self.packages_deps = packages_deps
+        self.store = store
+        self.wix_config = wix_config
+        self.paths = paths
+        self._parse_sources()
+        self._add_include()
+        self._customize_ui()
+
+    def _parse_sources(self):
+        sources_path = self.package.resources_wix_bundle
+        etree.register_namespace('bal', BAL_SCHEMA)
+        tree = etree.parse(sources_path)
+        self.root = tree.getroot()
+        for element in self.root.iter():
+            tag = element.tag[element.tag.index('}') + 1:]
+            if UTIL_SCHEMA in element.tag:
+                tag = 'util:' + tag
+            elif BAL_SCHEMA in element.tag:
+                tag = 'bal:' + tag
+            element.tag = tag
+        self.root.set('xmlns', WIX_SCHEMA)
+        self.root.set('xmlns:util', UTIL_SCHEMA)
+        self.root.set('xmlns:bal', BAL_SCHEMA)
+        self.product = self.bundle = self.root.find(".//Bundle")
+
+
+    def _add_include(self):
+        if self._with_wine:
+            self.wix_config = to_winepath(self.wix_config)
+        inc = etree.PI('include %s' % self.wix_config)
+        self.root.insert(0, inc)
+
+    def _customize_ui(self):
+        pass
+        # # Banner Dialog and License
+        # for path, var in [(self.BANNER_BMP, 'BannerBmp'),
+        #         (self.DIALOG_BMP, 'DialogBmp'),
+        #         (self.LICENSE_RTF, 'LicenseRtf')]:
+        #     path = self.package.relative_path(path)
+        #     if self._with_wine:
+        #         path = to_winepath(path)
+        #     if os.path.exists(path):
+        #         etree.SubElement(self.product, 'WixVariable',
+        #                 Id='WixUI%s' % var, Value=path)
+        # # Icon
```

Comment:
It was commented because I wasn't sure how the UI was going to be. Now that the UI is almost finished, this can go away.